src/Cli/TableFormatter.php

```diff
@@ -131,12 +152,23 @@ public function format($columns, $texts, $colors = [])
         $output = '';
         for ($index = 0; $index < $maxLength; $index++) {
             foreach ($columns as $column => $width) {
+                if ($this->isBorderedTable && $column === 0) {
+                    $output .= ltrim($this->border);
+                    $width = $width - 2;
```

Comment:
Adding padding and avoiding magic numbers.
```suggestion
                    $output .= $this->border . str_repeat(' ', $this->padding);
                    $width = $width - strlen($this->border) - $this->padding;
```